db/migrations/20191205180116_add_playlist_dashboard_page_user_migration.php

```diff
@@ -30,26 +30,25 @@ class AddPlaylistDashboardPageUserMigration extends AbstractMigration
     /** @inheritdoc */
     public function change()
     {
-        $pagesTbl = $this->table('pages');
-        $pagesTbl
-            ->insert([
-                'name' => 'playlistdashboard',
-                'title' => 'Playlist Dashboard',
-                'asHome' => 1
-            ])->save();
-
-        $groupTbl = $this->table('group');
-        $groupTbl->insert([
-            ['group' => 'Playlist Dashboard User', 'isUserSpecific' => 0, 'isEveryone' => 0, 'isSystemNotification' => 0],
-        ])->save();
-
-        $this->execute(
-            'INSERT INTO `permission` (`entityId`, `groupId`, `objectId`, `view`) 
-                    SELECT  1, 
-                    (SELECT groupId FROM `group` WHERE `group`.group = \'Playlist Dashboard User\'), 
-                    (SELECT pageId FROM `pages` WHERE `pages`.name = \'playlistdashboard\'), 
-                    1
-                ');
+
+        $page = $this->execute(
+            'INSERT INTO `pages` SET `name`=\'playlistdashboard\', `title`= \'Playlist Dashboard\', `asHome`=1;
+                   ');
+        $pageId = $this->getAdapter()->getConnection()->lastInsertId();
+
+        $group = $this->execute(
+            'INSERT INTO `group` SET `group`=\'Playlist Dashboard User\', `isUserSpecific`= 0, `isEveryone`= 0, `isSystemNotification`= 0;
+                   ');
+        $groupId = $this->getAdapter()->getConnection()->lastInsertId();
+
+        // Set Playlist Dashboard Page Permission
+        $this->execute('INSERT INTO `permission` (`entityId`, `groupId`, `objectId`, `view`) SELECT  1, '.$groupId.', '.$pageId.', 1');
```

Comment:
@PeterMis - I can't remember whether we can rely on these IDs or if we should look them up - I think look them up?

Reply:
entityId for Page entity in permissionentity is set to 1 on install, I think that should be fine.
However it might be better to be safe than sorry and look it up, just in case :).